### PR TITLE
Marketplace: Use mermaid to diagram User Journey and H/CI

### PR DESCRIPTION
**Pull request by KellyAH**
A diagram for info added in https://github.com/zinc-collective/convene/pull/1061 based off [rough sketch](https://github.com/zinc-collective/convene/pull/1061#discussion_r1083233119).

### CODE REVIEW

File: app/furniture/marketplace/README.md [hunk 1]
```diff
@@ -4,7 +4,9 @@
 
 ### Shopper Purchases Products
 
-<!-- it would be cool if someone could Mermaid this -->
+<!-- Chart created with Mermaid see: docs/checkout_flow_diagram_mermaid.txt -->
+
+![](../../../docs/checkout_flow_diagram.png)
```
Comment (KellyAH) [inline app/furniture/marketplace/README.md:9]: If you hate how the chart is dark, here's another color option, which IMHO is worse.
mermaid-diagram-2023-01-21-214228

Comment (zspencer) [reply app/furniture/marketplace/README.md:9]: FYI, Github's Markdown renderer supports Mermaid! You can put the Mermaid code inline in markdown files, so you don't need to compile the png file!

**Review by zspencer — APPROVED** [review]
I'm down for merging as is, but I would love to not need to generate a png or anything, and just have the mermaid chart definition inline!

File: app/furniture/marketplace/README.md (hunk 1 above)
Comment (KellyAH) [reply app/furniture/marketplace/README.md:9]: weird I tried that and it only displayed the code as code. Not as a chart.

Comment (KellyAH) [reply app/furniture/marketplace/README.md:9]: Screen Shot 2023-01-22 at 9 49 03 AM

Comment (zspencer) [reply app/furniture/marketplace/README.md:9]: Did you enclose it in backticks with:


 `````
 ```mermaid
 
 ```
 `````
 
 Per https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/?

Comment (KellyAH) [reply app/furniture/marketplace/README.md:9]: yes.
Screen Shot 2023-01-22 at 9 56 27 AM

Comment (zspencer) [reply app/furniture/marketplace/README.md:9]: This is the view from inside your editor; not github.com's UI. You will likely need to install a VS Code Extension for the Markdown previewer to support mermaid directly.

Comment (KellyAH) [reply app/furniture/marketplace/README.md]: Fixed! It [renders](https://github.com/zinc-collective/convene/blob/f6cf6540bd4cca4166b6f86045a7fd3debe682a6/app/furniture/marketplace/README.md)!

**Pull request merged**